.gitmodules

```diff
@@ -95,3 +95,67 @@
 	path = third_party/mbed-os-posix-socket/repo
 	url = https://github.com/ARMmbed/mbed-os-posix-socket.git
 	branch = main
+[submodule "p6/abstraction-rtos"]
```

Comment:
Could your sdk package be regrouped in one submodule?

Reply:
@jmartinez-silabs : thanks for your comments.
Yes.. we will look into this as part of next update.